dist/package.json

```diff
@@ -36,6 +36,7 @@
         "@angular/compiler": "13.2.6",
         "@angular/compiler-cli": "13.2.6",
         "@ngtools/webpack": "13.2.6",
+        "@types/mocha": "^10.0.0",
```

Comment:
^>:(

Reply:
Dist folder :( need to look into building the library one time. Once at a standup we discussed to do this later. We can't simply rebuild the entire dist folder for now